firmware/Core/Inc/telecommands/telecommand_parser.h

```diff
@@ -10,6 +10,10 @@
 
 #define TCMD_PREFIX_STR "CTS1+"
 #define TCMD_PREFIX_STR_LEN 5
+#define TCMD_HEX_ARRAY_LEN 32
+
+// TODO: Before launch, change value to 1
+static const uint8_t SHA256_ENABLED = 0;
```

Comment:
Please change this to a configuration variable, set to disabled by default. The configuration variable should be controllable by the general configuration variable scheme. Please name the variable better (e.g., `TCMD_require_valid_sha256`).

Reply:
Should be changed now.

Reply:
Updated this now, the variable is now configurable through the general scheme.